server/planning/events/events_utils.py

```diff
@@ -203,7 +213,17 @@ async def get_recurring_timeline(
 
     # Make sure we are working with a datetime instance
     if not isinstance(selected_start, datetime):
-        selected_start = datetime.strptime(selected_start, "%Y-%m-%dT%H:%M:%S%z")
+        try:
+            selected_start = datetime.strptime(selected_start, "%Y-%m-%dT%H:%M:%S%z")
+        except ValueError:
+            # Fallback for strings without timezone information:
+            selected_start = datetime.strptime(selected_start, "%Y-%m-%dT%H:%M:%S")
+            # If there's a timezone field in the event, use it:
+            tz_str = selected.get("dates", {}).get("tz")
+            if tz_str:
+                selected_start = pytz.timezone(tz_str).localize(selected_start)
+            else:
+                selected_start = selected_start.replace(tzinfo=pytz.UTC)
```

Comment:
I think there is chance of another `ValueError` if for some reason the format does not match with the date string. I would suggest using `arrow` as it is already used in the project. Something like this `arrow.get(selected_start).datetime` would do instead of the `datetime.strptime` that's inside the except. What do you think?